.github/workflows/test.yml

```
@@ -7,18 +7,18 @@ jobs:
         with:
           lfs: true
 
       - name: Set up Go
-        uses: actions/setup-go@v1
+        uses: actions/setup-go@v5
         with:
-          go-version: 1.17.x
+          go-version: 1.24.x
 
       - name: Set up Python
         uses: actions/setup-python@v1
```

actionlint (1.7.4)

the runner of "actions/setup-python@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)